lib/Doctrine/ORM/Tools/Console/Command/GenerateProxiesCommand.php

```diff
@@ -98,20 +89,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             );
         }
 
-        if ( count($metadatas)) {
-            foreach ($metadatas as $metadata) {
-                $output->writeln(
-                    sprintf('Processing entity "<info>%s</info>"', $metadata->name)
-                );
-            }
-
-            // Generating Proxies
-            $em->getProxyFactory()->generateProxyClasses($metadatas, $destPath);
+        if (empty($metadatas)) {
+            $ui->success('No Metadata Classes to process.');
+            return;
+        }
 
-            // Outputting information message
-            $output->writeln(PHP_EOL . sprintf('Proxy classes generated to "<info>%s</INFO>"', $destPath));
-        } else {
-            $output->writeln('No Metadata Classes to process.');
+        foreach ($metadatas as $metadata) {
+            $ui->text(sprintf('Processing entity "<info>%s</info>"', $metadata->name));
         }
+
+        // Generating Proxies
+        $em->getProxyFactory()->generateProxyClasses($metadatas, $destPath);
+
+        // Outputting information message
+        $ui->newLine();
+        $ui->text(sprintf('Proxy classes generated to "<info>%s</INFO>"', $destPath));
```

Comment:
Any reason why this one and the above are left out?

Reply:
I don't get what you mean

Reply:
I mean those `<info>` tags, I thought whole point of this PR is to eliminate/replace them with SymfonyStyle.

Reply:
we keep them, because the dest path should be green, and the rest of the text white